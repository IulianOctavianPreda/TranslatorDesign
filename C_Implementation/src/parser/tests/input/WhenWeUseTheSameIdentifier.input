bool aaa() {
    int aa;
    bool aa;
}
