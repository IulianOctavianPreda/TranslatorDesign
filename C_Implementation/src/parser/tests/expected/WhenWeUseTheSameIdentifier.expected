Program (1)
  Program (1)
  FnDecl (1)
    Type (1)
      TYPE: bool
    Id (1)
      ID: aaa
    Parameters (1)
      LRB
      RRB
    Block (1)
      LB
      DeclList (1)
        DeclList (1)
          DeclList (1)
          VarDecl (2)
            Type (2)
              TYPE: int
            Id (2)
              ID: aa
            SEMICOL
        VarDecl (3)
          Type (3)
            TYPE: bool
          Id (3)
            ID: aa
          SEMICOL
      StmtList (4)
      RB
Same identifier used - 'aa' lines:2, 3 
Same identifier used - 'aa' lines:3, 2 
